src/__tests__/pretty-dom.js

```diff
@@ -45,16 +45,34 @@ test('prettyDOM defaults to document.body', () => {
   </body>"
 `
   renderIntoDocument('<div>Hello World!</div>')
-  expect(prettyDOM()).toMatchInlineSnapshot(defaultInlineSnapshot)
-  expect(prettyDOM(null)).toMatchInlineSnapshot(defaultInlineSnapshot)
+  expect(prettyDOM()).toMatchInlineSnapshot(
```

Comment:
Hmmm... not sure what happened here, I need to investigate this change.

Reply:
Okay, fixed, it was a problem with automatically updating inline snapshots and then the previous expectation had the extra double quotes.